go/README.md

```diff
@@ -1 +1,200 @@
-# Go
\ No newline at end of file
+# Go Code Conventions
+
+## Modules
+
+go mod init bitbucket.org/aanzeeonline/[yourprojectname]
+
+Working with a local fork of a module, put an extra line in the go.mod:
+// replace bitbucket.org/aanzeeonline/libgoaz => ../LibGOAZ
+(and remove the comments)
+
+Before pushing the final update to git, make sure that the fork isn't in use anymore
+
+We need to investigate, if a vendor option (go mod vendor) is usable.
+(The upcoming 1.13 will have the final module solution.)
+"go mod vendor puts them in a vendor folder inside your project, running go build -mod vendor
+uses the dependency put in the vendor folder. This further isolates the project from the
+globally installed dependencies."
+
+## Project structure for an API / Web application
+
+Root folder
```

Comment:
Wellicht nog even goed kijken naar de MD structuur, leesbaar is het nu allerminst